addons/website_event_questions/views/event_views.xml

```diff
@@ -33,12 +33,12 @@
                             <tree>
                                 <field name="sequence" widget="handle" />
                                 <field name="title"/>
-                                <field name="is_mandatory_answer"/>
-                                <field name="once_per_order"/>
+                                <field name="is_mandatory_answer" string="Mandatory"/>
+                                <field name="once_per_order" string="Once per Order"/>
                                 <field name="question_type" string="Type" />
                                 <field name="answer_ids" widget="many2many_tags"
                                     attrs="{'invisible': [('question_type', '!=', 'simple_choice')]}" />
-                                <button name="action_view_question_answers" type="object" class="fa fa-bar-chart p-0" title="Answer Breakdown" />
+                                <button name="action_view_question_answers" type="object" class="fa fa-bar-chart p-0" string=" Stats" title="Answer Breakdown" />
```

Comment:
Does it need to be "Stats" ? We usually don't use shortened labels, but maybe there is not enough room.

There is a strange space before the word though, not sure if intended, you could try padding/margin classes to get the proper spacing instead?

Reply:
Hello @awa-odoo 
Yes the space is on purpose, otherwise the icon is stuck to the label and it is not clean. But not the cleanest way indeed, and not as frequent as i thought in the code, so I changed it to something more usual, style is a bit different.
Concerning the label, the idea is indeed to keep it clean and short. After talking with LNA, we can keep it that way :)
So it should be ok now, what do you think?